apps/src/music/progress/MusicValidator.ts

```diff
@@ -17,34 +17,102 @@ export interface ConditionNames {
 }
 
 export const MusicConditions: ConditionNames = {
-  PLAYED_SOUNDS_TOGETHER: {name: 'played_sounds_together', valueType: 'number'},
+  PLAYED_SOUNDS_TOGETHER: {
+    name: 'played_sounds_together',
+    valueType: 'number',
+    description:
+      'Successful when a minimum number of sounds were playing simultaneously. Ex. Value: 2',
+  },
   PLAYED_DIFFERENT_SOUNDS_TOGETHER: {
     name: 'played_different_sounds_together',
     valueType: 'number',
+    description:
+      'Successful when a minimum number of unique sounds were playing simultaneously. Ex. Value: 2',
```

Comment:
Maybe:
> Successful when at least this many unique sounds are playing simultaneously.

Reply:
This feels wordier to me? I'm leaving this for now, but I also asked Emma and Katie to weigh on the three options I could think of:

> Successful when a minimum number of sounds are playing simultaneously.
> Successful when when at least this many sounds are playing simultaneously.
> Successful when when at least the given number of sounds are playing simultaneously.

Reply:
The curriculum team agreed with your suggestion. Thanks!